config/helpers.php

```diff
@@ -434,6 +436,20 @@ function params(): array
 	}
 }
 
+if (Helpers::hasOverride('qr') === false) { // @codeCoverageIgnore
+	/**
+	 * Creates a QR code object
+	 */
+	function qr(string|ModelWithContent $data): QrCode
+	{
+		if ($data instanceof ModelWithContent) {
+			$data = $data->url();
+		}
+
+		return new QrCode($data);
```

Comment:
What speaks for moving the color options to the export methods is that they would become accessible to users of the helper and field methods as well.

Reply:
Yep, I get that argument. Makes sense to me.